Application/src/Utils/Color.js

```diff
@@ -0,0 +1,7 @@
+import ColorThief from 'colorthief';
+
+export function getColors(image) {
+	let ct = new ColorThief();
+	let result = ct.getColor(image.topLeft);
+	console.log(result);
```

Comment:
What is the purpose of logging the color result of the top left part of the image to the console?